### ISSUE TITLE
#%% not recognized in outline as cell separator

### ISSUE THREAD

**Issue by skjerns** [spyder-ide/spyder#14360]
Since the switch to LSP for the outline creation, `#%%` no longer shows up in the outline. I think this is not intended, or?

Example

```Python
#%% cell one
#%% cell two
```
![cell_outline](https://user-images.githubusercontent.com/14980558/100997039-19233a80-355a-11eb-98c2-a93318a01c00.gif)

## Versions

* Spyder version: 4.2.0 None
* Python version: 3.8.5 64-bit
* Qt version: 5.9.7
* PyQt5 version: 5.9.2
* Operating System: Windows 10

**Comment by andfoy** [comment]
Hi @skjerns, this issue was fixed in the latest version of pyls-spyder (v0.2.1), which you can install by using conda or pip:

```bash
# Anaconda (Recommended)
conda install pyls-spyder=0.2.1 -c spyder-ide

# Pip
pip install -U pyls-spyder
```

**Comment by skjerns** [comment]
perfect, thanks! didn't think of that
